packages/account/src/Sections/Verification/ProofOfOwnership/proof-of-ownership-form.jsx

```diff
@@ -0,0 +1,244 @@
+import classNames from 'classnames';
+import React, { useRef, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Button, useStateCallback } from '@deriv/components';
+import { Formik } from 'formik';
+import { localize } from '@deriv/translations';
+import FormFooter from '../../../Components/form-footer';
+import FormBody from '../../../Components/form-body';
+import FormSubHeader from '../../../Components/form-sub-header';
+import FormBodySection from '../../../Components/form-body-section';
+import { isMobile, readFiles, WS, DOCUMENT_TYPE } from '@deriv/shared';
+import Card from './card.jsx';
+import DocumentUploader from '@binary-com/binary-document-uploader';
+import { IDENTIFIER_TYPES, VALIDATIONS } from './constants/constants';
+
+const getScrollOffset = (items_count = 0) => {
+    if (isMobile()) return '200px';
+    if (items_count <= 2) return '0px';
+    return '80px';
+};
+const ProofOfOwnershipForm = ({
+    client_email,
+    grouped_payment_method_data,
+    refreshNotifications,
+    total_documents_required,
+    updateAccountStatus,
+}) => {
+    const grouped_payment_method_data_keys = Object.keys(grouped_payment_method_data);
+    const initial_values = {};
+    const [is_submit_button_disabled, setIsSubmitButtonDisabled] = React.useState(true);
+    const [form_state, setFormState] = useStateCallback({ should_show_form: true });
+    const form_ref = useRef();
+    const [document_upload_errors, setDocumentUploadErrors] = useState();
+    useEffect(() => {
+        setDocumentUploadErrors([]);
+    }, []);
+    const fileReadErrorMessage = filename => {
+        return localize('Unable to read file {{name}}', { name: filename });
+    };
+
+    const validateFields = values => {
+        const errors = {};
+        errors.data = [...document_upload_errors];
+        let total_documents_uploaded = 0;
+        let has_errors = false;
+        const cards = values?.data;
+        Object.keys(cards)?.forEach?.(card_key => {
+            const items = cards?.[card_key] ?? {};
+            errors.data[card_key] = errors.data?.[card_key] ?? [];
```

Comment:
just to double check: what we do on 49 line? if we have errors.data[card_key] we reassign it? and if we have no errors.data[card_key] we assign it to empty array?
what is the type of data for errors.data[card_key]?

Reply:
@yauheni-kryzhyk-deriv, `data[card_key]` can be `undefined` or it can contain an `object` with `{payment_identifier, files, ...}`. I do that, because I later reference the index to pick out the exact error to show for a given control. If I don't perform the check, I'll clear the previous errors, which is not desired

Reply:
@yauheni-kryzhyk-deriv, This is the example data that can be stored in `errors.data[0]`  [{ identifier: 'Please Complete This Field', files: ['Duplicate Upload']}]